pkg/chains/storage/docdb/docdb.go

```diff
@@ -262,6 +266,18 @@ func populateMongoServerURL(ctx context.Context, cfg config.Config) error {
 	logger := logging.FromContext(ctx)
 	mongoEnv := "MONGO_SERVER_URL"
 
+	if cfg.Storage.DocDB.MongoServerURLPath != "" {
+		logger.Infof("setting %s from storage.docdb.mongo-server-url-path: %s", mongoEnv, cfg.Storage.DocDB.MongoServerURLPath)
```

Comment:
It would be good to use constant `docDBMongoServerURLPathKey` instead of key directly

Reply:
I see its resolved but i don't see changes :thinking: